api/v1beta1/openstackcluster_types.go

```diff
@@ -219,7 +219,7 @@ type OpenStackClusterStatus struct {
 	APIServerLoadBalancer *LoadBalancer `json:"apiServerLoadBalancer,omitempty"`
 
 	// FailureDomains represent OpenStack availability zones
-	FailureDomains clusterv1.FailureDomains `json:"failureDomains,omitempty"`
+	FailureDomains []clusterv1.FailureDomain `json:"failureDomains,omitempty"`
```

Comment:
this is breaking change and it will require new API version. Using v1beta1 will not require this change.